lib/QubitFlatfileImport.class.php

```diff
@@ -962,6 +992,104 @@ private function setInformationObjectByKeymap($legacyId)
   }
 
   /**
+   * Handle various update options when importing repositories and actors.
+   *
+   * @return bool  Whether to skip row processing for this record.
+   */
+  private function handleRepositoryAndActorRow()
+  {
+    // Not updating and not skipping matches: create a new record without checking
+    if (!$this->isUpdating() && !$this->skipMatched)
+    {
+      $this->object = new $this->className;
+
+      return false;
+    }
+
+    // Check existing repo by auth. form of name
+    $query = "SELECT object.id
+      FROM object JOIN actor_i18n i18n
+      ON object.id = i18n.id
+      WHERE i18n.authorized_form_of_name = ?
+      AND object.class_name = ?;";
+
+    $statement = QubitFlatfileImport::sqlQuery($query, array($this->columnValue('authorizedFormOfName'), $this->className));
```

Comment:
I think we should use QubitPdo::* when possible for queries of this nature, sqlQuery is going to be removed eventually since it just copies the functionality and is only used in 2-3 places as opposed to QubitPdo which is everywhere.